app/models/users.py

```diff
@@ -0,0 +1,27 @@
+from sqlmodel import Field, SQLModel
+
+
+class Token(SQLModel):
+    """JSON Web Token (JWT) provided to the user after authentication."""
+
+    access_token: str
+    token_type: str
+
+
+class TokenData(SQLModel):
+    """TokenData links the JWT token to the username"""
+
+    username: str | None = None
```

Comment:
I'm sorry, however, I still don't understand the purpose of TokenData, why does this need to exist?
Will it be useful later?